spec/datetimeformat.html

```diff
@@ -500,7 +503,10 @@
           The array that is the value of the "nu" property of any locale property of [[LocaleData]] must not include the values "native", "traditio", or "finance".
         </li>
         <li>
-          [[LocaleData]][locale] must have hourCycle property with String values for all locale values.
+          [[LocaleData]][locale].hc must be the array [*null*, *"h11"*, *"h12"*, *"h23"*, or *"h24"*] for all locale values.
```

Comment:
Yes, the "or" is a typo.

I copied parts of the text of [[SearchLocaleData]][locale] (https://tc39.github.io/ecma402/#sec-intl-collator-internal-slots) for the [[LocaleData]][locale] description. And then I copied parts of [[LocaleData]][locale] for [[LocaleData]][locale].hc, but missed to remove the "or".